corehq/apps/export/esaccessors.py

```diff
@@ -24,7 +24,8 @@ def get_case_export_base_query(domain, case_type):
     return (CaseES(es_instance_alias=ES_EXPORT_INSTANCE)
             .domain(domain)
             .case_type(case_type)
-            .sort("opened_on"))
+            .sort("opened_on")
+            .sort('inserted_at', reset_sort=False))
```

Comment:
I patched it here instead of directly in the odata feed because I suspect the same issue could manifest in exports as well, it's just less likely since the page size is `5000` (there's a bit of chasing, but [the relevant line is here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/es_query.py#L500))